source/connect-to-mongo/stable-api.txt

```diff
@@ -58,10 +56,9 @@ The following code example shows how to specify {+stable-api+} version 1:
    :copyable:
    :dedent:
 
-Once you create a ``MongoClient`` instance with
-a specified API version, all commands you run with the client use the specified
-version. If you must run commands using more than one version of the 
-{+stable-api+}, create a new ``MongoClient``.
+Once you create a ``MongoClient`` instance with the {+stable-api+}, all commands you
+run with the client use the specified {+stable-api+} configuration. If you must run
+{+stable-api+} commands using alternative configurations, create a new ``MongoClient``.
```

Comment:
```suggestion
commands using alternative configurations, create a new ``MongoClient``.
```

Because you might want to run commands that are not part of the Stable API.